docs/notebooks/bold_realignment.ipynb

```diff
@@ -337,7 +337,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from nifreeze.model.base import AverageModel\n",
+    "from nifreeze.model.base import ExpectationModel\n",
```

Comment:
Average is too specific about the statistic used. If *Expectation* meets opposition, I'm happy to settle whatever others think is best (or even revert back to Average).